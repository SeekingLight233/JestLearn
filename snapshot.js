export const generateConfig = () => {
  return {
    server: 'http://localhost',
    port: '8080',
    domain: 'localhost',
    time: new Date(),
  };
};

export const generateConfig2 = () => {
  return {
    server: 'http://localhost',
    port: '8080',
    domain: 'localhost',
    time: new Date(),
  };
};
