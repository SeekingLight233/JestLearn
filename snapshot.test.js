import { generateConfig, generateConfig2 } from './snapshot';

// 测试配置文件的时候可以使用快照
test('测试 generateConfig', () => {
  expect(generateConfig()).toMatchSnapshot({
    time: expect.any(Date),
  });
});

test('测试 generateConfig', () => {
  expect(generateConfig2()).toMatchSnapshot({
    time: expect.any(Date),
  });
});
