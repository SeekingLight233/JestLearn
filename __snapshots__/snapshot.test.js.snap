// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试 generateConfig 1`] = `
Object {
  "domain": "localhost",
  "port": "8080",
  "server": "http://localhost",
  "time": Any<Date>,
}
`;

exports[`测试 generateConfig 2`] = `
Object {
  "domain": "localhost",
  "port": "8080",
  "server": "http://localhost",
  "time": Any<Date>,
}
`;
